src/_base/harness/config/functions.yml

```diff
@@ -219,3 +219,30 @@ function('version_compare', [version1, version2, operator]): |
   $version1 .= str_repeat('.0', max(0, $count2 - $count1));
   $version2 .= str_repeat('.0', max(0, $count1 - $count2));
   = version_compare($version1, $version2, $operator);
+
+function('bool', [value]): |
+  #!php
+  if (!is_bool($value)) {
+    echo 'deprecated: bool() will be removed in the future, so "' . $value . '" should be converted to a boolean before then'."\n";
```

Comment:
too annoying repeatedly saying it?

Reply:
Yes it will get annoying to see this message repeatedly.

I think it's fine as a note in the 2.0.0 UPGRADE.md entry.

Reply:
I'm not meaning this for a major version bump, the deprecation message is for the current major version